tests/scripts/123.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+This file is loaded automatically by Houdini at startup.
+"""
+import inspect
+import os
+import sys
+import hou
+
+current_file_path = os.path.abspath(inspect.getsourcefile(lambda: 0))
+
+sys.path.append(os.path.join(os.path.dirname(current_file_path), ".."))
+
+import run_tests  # noqa
```

Comment:
It seemed to make sense to keep the startup script and the run test logic separate so that the separation is clearer if we need to change the way the tests are invoked in Houdini later.